packages/backend/common/tasks.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger(__name__)
 
 
-class Task:
+class LambdaTask:
```

Comment:
Why `LambdaTask`?

Reply:
This is the class that launches AWS Lambda with EventBridge message.
I've introduced another type of tasks – Celery tasks, which means that we need to make the distinction obvious